backend/app/datamodels/restaurants.js

```diff
@@ -1,42 +1,42 @@
-const { Restaurants } = require("../db/models");
+const { Restaurants } = require('../db/models');
 
-const getRestaurantFromId = async (restaurantId) => {
-    const { _id, name: restaurantName, ownerName, maxInstances } = await Restaurants.findOne({ _id: restaurantId});
+const getRestaurantById = async (restaurantId) => {
+	const { _id, name: restaurantName, ownerName, maxInstances } = await Restaurants.findOne({ _id: restaurantId});
     
-    return {
-        restaurantId: _id,
-        restaurantName,
-        ownerName,
-        maxInstances
-      }
-  };
+	return {
+		restaurantId: _id,
+		restaurantName,
+		ownerName,
+		maxInstances
+	};
+};
 
-  const getRestaurantFromName = async (restaurantNameFromUrl) => {
-    const { _id, name: restaurantName, ownerName, maxInstances } = await Restaurants.findOne({ name: restaurantNameFromUrl});
+const getRestaurantByName = async (restaurantNameFromUrl) => {
+	const { _id, name: restaurantName, ownerName, maxInstances } = await Restaurants.findOne({ name: restaurantNameFromUrl});
     
-    return {
-        restaurantId: _id,
-        restaurantName,
-        ownerName,
-        maxInstances
-      }
-  };
+	return {
+		restaurantId: _id,
+		restaurantName,
+		ownerName,
+		maxInstances
+	};
+};
   
-  const getRestaurantFromHostname = async (hostname) => {
-    const { _id, name: restaurantName, ownerName, maxInstances } = await Restaurants.findOne({ externalHostnames: hostname })
+const getRestaurantByHostname = async (hostname) => {
+	const { _id, name: restaurantName, ownerName, maxInstances } = await Restaurants.findOne({ externalHostnames: hostname });
    
-    return {
-      restaurantId: _id,
-      restaurantName,
-      ownerName,
-      maxInstances
-    }
-    };
+	return {
+		restaurantId: _id,
+		restaurantName,
+		ownerName,
+		maxInstances
+	};
+};
 
 
-  module.exports = {
-    getRestaurantFromHostname,
-    getRestaurantFromId,
-    getRestaurantFromName
-  };
+module.exports = {
+	getRestaurantById,
+	getRestaurantByName,
+	getRestaurantByHostname,
+};
   
\ No newline at end of file
```

Comment:
The code patch is a refactoring of an existing module that exports three functions to retrieve restaurant data based on different parameters, namely ID, name and hostname.

The changes made in the patch seem reasonable and do not introduce any obvious bugs or risks. The variable names are clear and the formatting is consistent throughout the file. 

One minor suggestion for improvement could be to add some error handling in case the `Restaurants.findOne()` method returns nothing or throws an error. This could involve either wrapping the function calls in try/catch blocks or using Promise rejections.

Overall, this is a good example of clean code refactoring that improves readability and maintainability without introducing new problems.